### PR TITLE
feat: add global navigation (resolves #308)

### LINKED ISSUE #308: Global navigation bar
## Is your feature request related to a problem? Please describe.

We need a global navigation component to facilitate navigation between different Platform Co-op Development Kit sites.

## Describe the solution you'd like

https://github.com/platform-coop-toolkit/coop-library/issues/253

## Describe alternatives you've considered

Not applicable.

## Additional context

Not applicable.

### PR DESCRIPTION
**Pull request by greatislander**
* [x] I've read the [guidelines for Contributing to the Platform Co-op Toolkit](https://github.com/platform-coop-toolkit/.github/blob/master/CONTRIBUTING.md)
* [x] This isn't a duplicate of an existing pull request

## Description

Adds global navigation molecule.

## Steps to test

Review Global Navigation component.

## Additional information

Not applicable.

## Related issues

- Resolves #308.

### CODE REVIEW

**Comment by cherylhjli** [comment]
Screen Shot 2020-04-15 at 4 13 50 PM
It's looking a little funky for me!

**Comment by greatislander** [comment]
@cherylhjli fixed now

File: src/components/molecules/global-nav/global-nav.njk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<nav class="global-nav" aria-labelledby="global-nav-label">
+	<span id="global-nav-label" class="screen-reader-text">global navigation</span>
+	<ul class="global-nav__list">
+		<li class="global-nav__list-item global-nav__list-item--first">
+			<a class="global-nav__link" rel="external" href="https://platform.coop">{% render '@svg', {svg: 'pcc', viewBox: '0 0 32 32'}, true %} Platform Cooperativism Consortium</a>
+		</li>
+		<li class="global-nav__list-item">
+			<a class="global-nav__link" rel="current" href="https://resources.platform.coop/">Resource Library</a>
```
Comment (jhung) [inline src/components/molecules/global-nav/global-nav.njk:8]: `aria-current` is probably needed here.

**Comment by jhung** [comment]
@cherylhjli, I find the use of the pine tree logo twice confusing as they are visually similar and in roughly the same screen position (especially on mobile) - but do different things.

What do you think of the idea of putting the global navigation prominently above the footer?

**Comment by cherylhjli** [comment]
@jhung intended behaviour for mobile is that this global menu goes away, so I think what Ned has there is what is specced in the design. We tried a dropdown version, but it didn't quite work.

The resource library actually is supposed to have its own logo now to avoid the quadruple pine situation - it's in the v03 core screens file!

**Comment by greatislander** [comment]
> I find the use of the pine tree logo twice confusing as they are visually similar and in roughly the same screen position (especially on mobile) - but do different things.

This is addressed here: https://github.com/platform-coop-toolkit/pinecone/pull/310

File: src/components/molecules/global-nav/global-nav.njk (hunk 1 above)
Comment (greatislander) [reply src/components/molecules/global-nav/global-nav.njk]: I'm using `aria-current="true"` as opposed to `aria-current="page"`. For reference:

> `true` represents the current item within a set

https://tink.uk/using-the-aria-current-attribute/

**Pull request merged**